x/tokenomics/keeper/token_logic_modules_test.go

```diff
@@ -54,11 +55,23 @@ func TestProcessTokenLogicModules_HandleAppGoingIntoDebt(t *testing.T) {
 	keepers.SetApplication(ctx, app)
 
 	// Add a new supplier
+	supplierAddress := sample.AccAddress()
 	supplierStake := cosmostypes.NewCoin("upokt", math.NewInt(1000000))
 	supplier := sharedtypes.Supplier{
-		OwnerAddress: sample.AccAddress(),
-		Address:      sample.AccAddress(),
+		OwnerAddress: supplierAddress,
+		Address:      supplierAddress,
 		Stake:        &supplierStake,
+		Services: []*sharedtypes.SupplierServiceConfig{
+			{
+				Service: service,
+				RevShare: []*sharedtypes.ServiceRevShare{
```

Comment:
Are we guaranteed there will always be a rev share?

Perhaps a default needs to be set in the "staking server" business logic + the necessary tests?

Reply:
I'll update this by making the stake config parser attribute a default value if none is found (UX), while making `ValidateServiceRevShare` which is also used by `MsgStakeSupplier.ValidateBasic` ensure that there is at least one entry (no implicit value logic)

Reply:
The staking config parser already attributes default values before building the `MsgStakeSupplier`.

I'll ensure that its `ValidateBasic` ensures that values exist.